CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the vscode-dbt-power-user extension! We greatly appreciate the dedication of our community members in helping us enhance this project.
+
+## Bug Reports and Feature Requests
+
+One of the simplest ways to contribute is by sharing your thoughts with us. If you come across any bugs or have ideas for new features or improvements, please start a discussion in our dedicated channel: `#tools-dbt-power-user` on the [dbt community Slack](https://www.getdbt.com/community/join-the-community/). Additionally, take a look at the [GitHub issues](https://github.com/innoverio/vscode-dbt-power-user/issues) to see if the topic has already been raised. If not, feel free to create a new issue. Your detailed bug reports, along with steps to reproduce the issue and relevant environment information, are invaluable to us and other contributors.
+
+## How to Contribute
+
+We warmly welcome contributions from our active community! If you're interested in contributing to the extension, follow these steps:
+
+1. **Fork the Repository:** Start by forking the repository to your own GitHub account.
+
+2. **Clone the Repository:** Clone your forked repository to your local machine:
+
+   ```bash
+   git clone https://github.com/your-username/vscode-dbt-power-user.git
+   cd vscode-dbt-power-user
+   ```
+
+3. **Set Up Development Environment:** Setting up the development environment for the vscode-dbt-power-user extension is designed to be straightforward. Most of the setup is already prepared, but you might need to follow a few steps to ensure a smooth experience:
+
+   1. **Install Node.js and npm:** If you haven't already, make sure you have Node.js and npm (Node Package Manager) installed on your machine. You can download and install them from the [official Node.js website](https://nodejs.org/).
```

Comment:
Maybe we should add minimum versions as well? I'm actually not totally sure which versions are supported.

Reply:
its difficult to estimate a minimum version. especially because you are not using a newer version-specific feature. most of the requirements will come from packages and they all seem to have older EOL-ed versions of node mostly. For now I'll add a note saying prefer the latest LTS version of node to get the most benefits of latest features and bugfixes